README.md

```diff
@@ -21,26 +21,66 @@ npm install sh-cmd-tag
 
 ## Quick Start
 
+### Basic command execution
+
 ```javascript
 import { sh, cmd } from "sh-cmd-tag";
 
-// Basic usage
 const result = await sh`echo "Hello World"`;
-console.log(result.output); // "Hello World"
+```
+
+The `result` variable contains a `ProcessResult` object with information about the command execution:
+
+```javascript
+{
+  ok: true,
+  output: "Hello World\n",
+  debug: ""
+}
+```
+
+### Safe interpolation of variables
 
-// Safe interpolation
+```javascript
 const filename = "my file.txt";
-await sh`touch ${filename}`; // Automatically escaped as 'my file.txt'
+await sh`touch ${filename}`;
+```
 
-// Object interpolation
+This automatically escapes the filename as `'my file.txt'`. The final command that gets executed is:
+
+```sh
+touch 'my file.txt'
+```
+
+### Object interpolation for command flags
+
+```javascript
 const config = { host: "localhost", port: 3000 };
```

Comment:
include an entry like "quiet": false to demonstrate the falsy values are dropped